components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -205,6 +211,10 @@ spec:
         ports:
         - containerPort: {{ .Values.global.servicePorts.dashboard }}
         volumeMounts:
+          {{ if .Values.global.tls.dashboard }}
+          - name: {{ .Values.dashboard.tls.secretName }}
+            mountPath: /etc/certs
```

Comment:
it looks okay but I have a couple of concerns. If we mount the secret directly to `/etc/certs`, the mount will directly overrides `/etc/certs` folder which seems fine while deploying the pod. However, what happens if users want to attach extra volumes to dashboard pod? Do you think will it cause a problem?

Reply:
You're totally right about the mechanism of overriding the specific folder but I don't see the need of having more files in the /etc/certs folder of the dashboard. If they need to mount smth they can use a different path. If you figure out a legitimate use case, we can indeed maybe move it one directory deeper and mount it there.

Reply:
Maybe we make it deeper from the get go to avoid those issues?

Reply:
Fixed this by adding `certificatesMountPath` under the 'tls' section of each component (dashboard, gateway)